core/server/api/v2/utils/serializers/output/slugs.js

```diff
@@ -0,0 +1,19 @@
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:slugs');
+const common = require('../../../../../lib/common');
+
+module.exports = {
+    generate(slug, apiConfig, frame) {
```

Comment:
The actions here are pre-defined:

- all
- browse
- add
- edit
- destroy

I know why the tests pass, because you have not updated the web routes. It still uses v0.1

Reply:
🤔 I am not sure I got all of it, so if actions here are pre-defined then this serialization happens inside controller directly or we can use `all`?

Reply:
> I know why the tests pass, because you have not updated the web routes. It still uses v0.1

If you change this, you will see what i mean :)
You could also take a look at the serializer handler in the shared package. It does not execute `generate`.

Reply:
yeah its not even using it! 🤦‍♂️

Reply:
My initial comment was wrong. The function name is dynamic, see serializer handler (docName + methodName). So `generate` could work.